esmvaltool/cmor/_fixes/CMIP6/IPSL_CM6A_LR.py

```diff
@@ -0,0 +1,38 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for IPSL-CM6A-LR."""
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Fixes for all vars."""
+
+    def fix_metadata(self, cubelist):
+        """
+        Removes addition cube in cube list (Cell area).
+        It has been created as a cell_area_cube which can be re-added here,
+        if needed.
+
+        Also forced the latitude and longitude to be 'lat' & 'lon',
+        instead of 'nav_lat' & 'nav_lon'.
+
+        Parameters
+        ----------
+        cubelist: iris CubeList
+            List of cubes to fix
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        for cube in cubelist:
+            if cube.standard_name == 'cell_area':
+                cell_area_cube = cube.copy()
```

Comment:
```suggestion
```
this line does nothing, can be removed

Reply:
Sorry, I  left this line here as a reminder. 

At the moment, the cell area and other data are loaded as a list of cubes. In this code, the cell area cube is deleted from the list, but  would it possible to add the cell area to the main data cube as metadata?